simple_network_sim/network_of_populations/__init__.py

```diff
@@ -26,7 +27,7 @@
 NodeName = str
 Time = int
 
-RESULT_DTYPES = {"time": "int16", "age": "category", "state": "category", "node": "category"}
+RESULT_DTYPES = {"date": str, "age": "category", "state": "category", "node": "category"}
```

Comment:
This is not a datetime because hdf5 does not accept any date/time format.

Reply:
Would an int with seconds since epoch be better?

Reply:
So that's how pd.Timestamp is stored, as a long (because it's in ns since epoch I believe). I've checked and it does not uses more memory if it's a string vs pd.Timestamp:
![image](https://user-images.githubusercontent.com/24376790/87934172-72487100-ca86-11ea-9b61-f254262ccad3.png)

Then we are left with making some more custom int format, per you example, we don't need seconds, we could use number of days since epoch, like Excel is doing. And convert through `pd.Timestamp(v, unit='D')`. But I'd be tempted to resort to this only if we see a real memory issue due to this, I haven't but I haven't ran very large networks either...

Reply:
Sounds good. I think the str dates are probably a lot more readable. You're right. Probably better leave them as strings.